### ISSUE TITLE
Support multi-scene OME-TIFF

### ISSUE THREAD

**Issue by anntzer** [filippocastelli/pyometiff#7]
The example dataset at https://docs.openmicroscopy.org/ome-model/6.0.0/ome-tiff/data.html#plate contains multiple (thousands of) scenes (think "plate positions").  aicsimageio allows accessing them via the `scenes` and `current_scene` attributes and `set_scene` method on the `AICSImage` object.  On the other hand, AFAICT (but I may be wrong), pyometiff only lets one access the first scene.  (My guess is that accessing the other scenes requires passing the `series` kwarg to `TiffFile.asarray(...)`.)

**Comment by filippocastelli** [comment]
you're absolutely right, thanks  for the feedback (I never actually tested on that stack).

It would be trivial to add a `series` arg to open different series, but I'd need a moment to evaluate how to integrate multi-series support at large.
I'll take a look ASAP but if you have suggestions I'm more than happy to receive them!
Thanks again
